examples/pubsub/main.tf

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  credentials = "${file(var.credentials_path)}"
+}
+
+# Example of an organization-level sink to Pub/Sub topic
+module "org_sink" {
+  source = "../../"
+  name   = "test-org-sink-pubsub"
+  org_id = "${var.org_id}"
+  filter = "severity > WARNING"
+
+  pubsub = {
+    name    = "org-sink"
+    project = "${var.alt_project_name}"
+  }
+}
+
+# Example of an organization-level sink to Pub/Sub topic
```

Comment:
Remove commented-out code.

Reply:
Ok. Was keeping this to add support for billing-level sinks later. I'll keep it in a branch instead.